addons/mail/models/mail_channel.py

```diff
@@ -58,6 +58,12 @@ def _get_default_image(self):
         help="Members of those groups will automatically added as followers. "
              "Note that they will be able to manage their subscription manually "
              "if necessary.")
+    message_unread = fields.Boolean(
+        'Unread Messages', compute='_get_message_unread',
```

Comment:
`_get_message_unread` --> `_compute_message_unread`

Reply:
delete message_unread

Reply:
I would keep the generic one as it is useful in a daily use.